### PR TITLE
[v2] Make Gene Searchable

### PR DESCRIPTION
**Pull request by alloy**
```graphql
{
  searchConnection(query: "minim", first: 10, entities: [GENE], mode: AUTOSUGGEST) {
    edges {
      node {
        ... on Gene {
          name
        }
      }
    }
  }
}
```

```json
{
  "data": {
    "searchConnection": {
      "edges": [
        {
          "node": {
            "name": "Minimalism"
          }
        },
        {
          "node": {
            "name": "Minimalism and Contemporary Minimalist"
          }
        },
        {
          "node": {
            "name": "Contemporary Minimalist"
          }
        },
        {
          "node": {
            "name": "Political Minimalism"
          }
        },
        {
          "node": {
            "name": "Postminimalism"
          }
        }
      ]
    }
  }
}
```

### CODE REVIEW

**Comment by alloy** [comment]
@mzikherman We should probably follow-up with either making gene batchable or see if Reaction can’t rely on just the data already included with a `SearchableItem`, as this will fetch a gene for each entry. What do you think?

File: src/schema/v2/search/SearchResolver.ts [hunk 1]
```diff
@@ -23,16 +23,23 @@ export class SearchResolver {
...
       Artist: artistLoader,
       Artwork: artworkLoader,
+      Gene: geneLoader,
     }
 
     const loader = loaderMapping[searchResultItem.label]
 
     if (loader) {
       return loader(searchResultItem.id)
+    } else {
```
Comment (sepans) [inline src/schema/v2/search/SearchResolver.ts]: 👍

**Comment by sepans** [comment]
Cool, Need to go deeper into `SearchResolver` to understand the magic it does but LGTM.

**Comment by sepans** [comment]
@mzikherman sorry I didn't wait for your response 😞

**Comment by alloy** [comment]
@sepans No worries, this change is still good as this field _can_ yield genes as search results, but nonetheless we can decide to use the more efficient results in Reaction.

**Pull request merged**